app/controllers/api/V1/greetings_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.find(rand(1..5))
+
+    render json: @greetings
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- - Project requirement:
 
![Selection_018](https://github.com/CesarHerr/hello-rails-back-end/assets/107632780/24717a15-fc06-4a87-b594-0da927034058)

- [ ] Your web app `both front-end and back-end` works as expected but as stated in the project requirement, you need to create 5 random greetings that will be displayed. You need to do this in the `seeds.rb` file. This will help add the greetings to the database when you run `rails db:seed` as there is no UI interface to do that yet. I'll suggest you implement this. :wave: :smile: :small_airplane:  This is a snippet of how you can do that:
```ruby
Greeting.create(greetings: 'Zdravstvuyte, Comrade')
```